### ISSUE TITLE
BJD_tdb to JD_utc conversion tool

### ISSUE THREAD

**Issue by trevorfoote** [PandoraMission/pandora-scheduler#2]
Within transits.py current conversion of time from BJD_tdb to MJD_utc includes the use of barycorr.py which requires querying the website: https://astroutils.astronomy.osu.edu/time/bjd2utc.html 
This is not ideal as it requires our computer to access the internet to run the code, requires that the website being queried is up and running properly, and that the website does not change over time.

Future work: A long-term fix requires the creation of a stand-alone BJD_tdb to JD_utc converter that can be added to our package to replace barycorr.py and remove the need to query a website. Such code exists as part of the EXOFAST package (https://github.com/jdeast/EXOFASTv2) but it is in IDL so we will need to update the code from IDL to Python before we can replace the existing converter.
